code/game/objects/items/circuitboards/machine_circuitboards.dm

```diff
@@ -141,7 +141,6 @@
 /obj/item/circuitboard/machine/tesla_coil
 	name = "Tesla Controller (Machine Board)"
 	greyscale_colors = CIRCUIT_COLOR_ENGINEERING
-	desc = "You can use a screwdriver to switch between Research and Power Generation."
```

Comment:
Surely you can give it something for a description

Reply:
Radiation collector circuits don't have a description either, if I think of something suitably witty I'll add it

Reply:
"Does not let you shoot lightning from your hands"